mlir/test/lib/Conversion/ConvertToSPIRV/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+# Exclude tests from libMLIR.so
+add_mlir_library(MLIRTestSPIRVFuncSignatureConversion
```

Comment:
Should we call this library after the umbrella conversion pass `ConvertToSPIRV` vs. this specific test pass?

Reply:
Renamed library to `MLIRTestConvertToSPIRV`